README.md

```diff
@@ -617,6 +617,9 @@ notify:
 # Global settings for all probes and notifiers.
 settings:
 
+  # Daemon settings
+  pid: /var/run/easeprobe.pid # pid file path, default: $CWD/easeprobe.pid
```

Comment:
There should be an option to disable pid file writing all together if needed. This will make easeprobe easier to run on read-only containers

Reply:
thanks for this input. If we want to disable the pid file, just configure the empty string.